src/features/search/searchSlice.js

```diff
@@ -56,6 +59,6 @@ export const {
 export const selectState = state => state.search;
 export const selectMovies = state => state.search.moviesList;
 export const selectSearchPeople = state => state.search.peopleList;
-export const selectSearchMoviesStatus = state =>state.search.statusMovie;
-export const selectSearchPeopleStatus = state =>state.search.statusPeople;
+export const selectSearchMoviesStatus = state => state.search.statusMovie;
+export const selectSearchPeopleStatus = state => state.search.statusPeople;
 export default searchSlice.reducer;
```

Comment:
We will have to work on the names of the selectors, but that's a task for the next sprint

Reply:
Yep, we have to do this on the next sprint